WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -210,6 +206,9 @@ import WordPressComAnalytics
         // pull to refresh animation.
         view.userInteractionEnabled = readerTopic != nil
 
+        NSNotificationCenter.defaultCenter().addObserver(self, selector: #selector(defaultAccountDidChange(_:)), name: WPAccountDefaultWordPressComAccountChangedNotification, object: nil)
```

Comment:
Certainly no harm leaving it as is.  We can always revisit after the split view work is done.